src/pages/dashboard/dashboard.ts

```diff
@@ -0,0 +1,48 @@
+import { Component } from '@angular/core';
+import { IonicPage, NavController, NavParams } from 'ionic-angular';
+// import { Chart } from 'chart.js';
+
+
+import { MainPage } from '../../shared/main-page.interface';
+import Pages from '../pages';
+
+
+/**
+ * Generated class for the DashboardPage page.
+ *
+ * See https://ionicframework.com/docs/components/#navigation for more info on
+ * Ionic pages and navigation.
+ */
+
+@IonicPage()
+@Component({
+  selector: 'page-dashboard',
+  templateUrl: 'dashboard.html',
+})
+export class DashboardPage implements MainPage {
+
+  // Doughnut
+  public doughnutChartLabels: string[] = ['Month', 'Week', 'Today'];
```

Comment:
`doughnutChartLabels` name could be renamed to something that depicts what that chart is meant to show.